src/tests/fake/user.tsx

```diff
@@ -101,3 +102,18 @@ export function mockApiFindUsers(users: Array<User>, query?: Query) {
             items: users,
         });
 }
+
+/**
+ * Mocks an API call for getting the user details
+ *
+ * @param userId - The users unique id
+ * @param userDetail - The details of the user
+ * @returns A nock scope for the mocked API call
+ */
+export function mockApiGetUser(userId: string, userDetail: User) {
+    return nock("http://localhost").get(`/api/users/${userId}`).reply(200, userDetail);
+}
+
+export function mockApiEditUser(userId: string, statisCode: number, update: any) {
```

Comment:
```suggestion
export function mockApiEditUser(userId: string, statusCode: number, update: any) {
```
? Also I think the return type on editing a user might be UserDetailed. We don't leverage that in the UI, but migt as well reflect what the API does